code/datums/gamemode/factions/bloodcult/bloodcult_spells.dm

```diff
@@ -264,6 +264,9 @@ var/list/uristrune_cache = list()
 
 /spell/cult/blood_dagger/cast(var/list/targets, var/mob/living/carbon/user)
 	..()
+	if (user.muted())
```

Comment:
no muted() check on trace and erase rune?

Reply:
There is on trace rune, it's in before_channel()

I figured I could keep them able to erase runes.